src/forms/TimeInput.jsx

```diff
@@ -2,6 +2,27 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
 
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import SelectInput from './SelectInput';
+
+export const HOURS_INPUT_CLASS = 'timeInput-hours';
+export const MINUTES_INPUT_CLASS = 'timeInput-minutes';
+export const MERIDIAN_INPUT_CLASS = 'timeInput-meridian';
+export const HIDDEN_INPUT_CLASS = 'timeInput-hidden';
+
+const formatDigits = (number) => `0${number}`.slice(-2);
+const getTimeParts = (time, part) => {
+	const splitTime = time.split(':');
+	const timeParts = new Object();
```

Comment:
just assign the whole object in one go.

```js
const [ hours, minutes ] = time.split(':');
const parts = {
  hours,
  minutes,
  meridian: hours < 12 ? 'AM': 'PM',
};
return parts[part];
```

also, avoid `new Object()` - use the `{}` object literal.